jmclient/jmclient/blockchaininterface.py

```diff
@@ -66,6 +72,13 @@ def fee_per_kb_has_been_manually_set(self, N):
         else:
             return False
 
+    def post_sync_wallet_callback(self, wallet):
```

Comment:
The idea of the WalletService architecture is to let the WalletService object be responsible for managing the state of (and interconnection between) the wallet and the blockchain interface. I would strongly prefer to put this callback in the WalletService.

Note I wouldn't make that kind of comment about passing wallet_name arguments into bci of course, because they're arguments to the queries we make. So it's basically arguing for bci to be just a wrapper around rpc (or whatever other tech) queries to a source of blockchain data and not managing any state otherwise.

I realise this case is pretty trivial, just want to make it clear why I pick up on it - don't want to update wallet state with a bci method.

Reply:
Done